### ISSUE TITLE
report bugs 

### ISSUE THREAD

**Issue by Rafaelfilp** [qingjun1991/GameDreamFactory#4]
please fix the android 12 problem when entering mugen it closes, and the infinite jump bug and when running the dummy run very fast thanks in advance

**Comment by qingjun1991** [comment]
1、在我国，有很多粉丝用的就是安卓12，他们都能正常玩，请你排查一下是否给了apk权限，!GameDreamFactory/log.txt，再看看这个文件的提示，看不懂的话，发给我看看
2、这个可能是跟人物包有关系，目前我没发现有哪个人物有这种bug，请你把人物包发给我看看

**Comment by Rafaelfilp** [comment]
The bug in Android 12 has now been resolved the only annoying bug and the character runs very fast and when I jump the character goes upstairs and doesn't come back if he wants to send a video or the character that is Occurring the bug 

Link from char:https://mugenarchive.com/forums/downloads.php?do=file&id=165703-blue-v-rugal--bernard-f-

**Comment by Rafaelfilp** [comment]
thank you very much, I always wanted to play mugen on my cell phone but it is weak and does not run exagear

**Comment by qingjun1991** [comment]
> The bug in Android 12 has now been resolved the only annoying bug and the character runs very fast and when I jump the character goes upstairs and doesn't come back if he wants to send a video or the character that is Occurring the bug
> 
> Link from char:https://mugenarchive.com/forums/downloads.php?do=file&id=165703-blue-v-rugal--bernard-f-
这个人物在我的手机上没发现你说的问题，我怀疑是文件乱码导致的问题，你可以自己排查一下：
1、确保你用的是最新版本的程序，"!GameDreamFactory"也必须是最新的
2、确保除了"!GameDreamFactory\data\select.def"，"!GameDreamFactory"里面的所有文件都没修改过，且没有换过
3、如果还不正常，把你这个人物的所有文本文件，用shift_jis编码打开，再保存成utf8带bom编码的文件

**Comment by qingjun1991** [comment]
> The bug in Android 12 has now been resolved the only annoying bug and the character runs very fast and when I jump the character goes upstairs and doesn't come back if he wants to send a video or the character that is Occurring the bug
> 
> Link from char:https://mugenarchive.com/forums/downloads.php?do=file&id=165703-blue-v-rugal--bernard-f-

这个人物在我的手机上没发现你说的问题，我怀疑是文件乱码导致的问题，你可以自己排查一下：
1、确保你用的是最新版本的程序，"!GameDreamFactory"也必须是最新的
2、确保除了"!GameDreamFactory\data\select.def"，"!GameDreamFactory"里面的所有文件都没修改过，且没有换过
3、如果还不正常，把你这个人物的所有文本文件，用shift_jis编码打开，再保存成utf8带bom编码的文件

**Comment by qingjun1991** [comment]
> > > O bug no Android 12 já foi resolvido o único bug chato e o personagem roda muito rápido e quando eu pulo o personagem sobe e não volta se quer mandar um video ou o personagem que está ocorrendo o bug
> > > Link do char: https://mugenarchive.com/forums/downloads.php?do=file&id=165703-blue-v-rugal--bernard-f-
> > 
> > 
> > 这个人物在我的手机上没发现你说的问题,我怀疑是文件乱码导致的问题，你可以自己排查一下: 1、确保你用的是最新版本的程序，"!GameDreamFactory"也必须是最新 的 2、确保除了"!GameDreamFactory\data\select.def"，"!GameDreamFactory"里面的所有文件都没修改过，且没有换过3、如果 还不正常，把你这个人物的所有文本文件，用shift_jis编码打开, 再保存成utf8带bom编码的文件
> 
> Could you send me a video of this process?

我想到了一个更关键的问题，你是不是修改了这个人物，如果你没有选对编码，就会导致app读取出错

**Comment by Rafaelfilp** [comment]
> > > > O bug no Android 12 já foi resolvido o único bug chato e o personagem roda muito rápido e quando eu pulo o personagem sobe e não volta se quer mandar um video ou o personagem que está ocorrendo o bug Link do char: https:// 
> > > > mugenarchive [. com/forums/downloads.php?do=file&id=165703-blue-v-rugal--bernard-f-](https://mugenarchive.com/forums/downloads.php?do=file&id=165703-blue-v-rugal--bernard-f-)
> > > 
> > > 
> > > 这个人物在我的手机上没发现你说的问题,我怀疑是文件乱码导致的问题，你可以自己排查一下: 1、确保你用的是最新版本的程序，"!GameDreamFactory"也必须是最新 的2、确保除了"!GameDreamFactory\data\select.def"，"!GameDreamFactory"里面的所有文件都没修改过，且没有换过3、如果 还不正常，把你这个人物的所有文本文件，用shift_jis编码打开, 再保存成utf8带bom编码的文件
> > 
> > 
> > Você poderia me enviar um vídeo desse processo?
> 
> 我想到了一个更关键的问题，你是不是修改了这个人物，如果你没有选对编码，就会导致app读取出错

I didn't modify

**Comment by Rafaelfilp** [comment]
> 



> 我想不出还有什么能导致出现这种问题；在我手机无法重现这个问题，就难了；你能否录个视频，从启动游戏到出现问题的过程，发到电子邮箱： 1257526872 [@ qq.com](mailto:1257526872@qq.com) , 我看看是不是哪些细节漏掉了

Of course I will do it now

**Comment by qingjun1991** [comment]
> 



> > 
> 
> > 我想不出还有什么能导致出现这种问题；在我手机无法重现这个问题，就难了；你能否录个视频，从启动游戏到出现问题的过程，发到电子邮箱： 1257526872 [@ qq.com](mailto:1257526872@qq.com) , 我看看是不是哪些细节漏掉了
> 
> Of course I will do it now

你视频中的人物，和你上面发给我的人物不是同一个？

**Comment by Rafaelfilp** [comment]
> 



> > 
> 
> > > 
> > 
> > 
> > > 我想不出还有什么能导致出现这种问题；在我手机无法重现这个问题，就难了；你能否录个视频，从启动游戏到出现问题的过程，发到电子邮箱： 1257526872 [@ qq.com](mailto:1257526872@qq.com) , 我看看是不是哪些细节漏掉了
> > 
> > 
> > claro que vou fazer agora
> 
> 你视频中的人物，和你上面发给我的人物不是同一个？

No, because what I sent above is automatic, it would not be possible to show the bug and this bug occurs with any character

**Comment by Rafaelfilp** [comment]
> 你发的新视频中，真是太神奇了，我需要花点时间研究下

Yes unfortunately this is preventing me from playing this bug occurs with any character I'll leave here the specifications of my device a03core 2gb 32ram, this bug occurs more on weaker cell phones

**Comment by qingjun1991** [comment]
问题已经解决，请更新补丁

**Comment by Rafaelfilp** [comment]
> 问题已经解决，请更新补丁

Now it's working normally thank you very much

**Comment by Rafaelfilp** [comment]
> 问题已经解决,请更新补丁

Good night, how are you? I hope so, I noticed a problem in dream factory, when installing the app on Android 11 it says that the app was made for an older version of Android and that it may not work correctly

A joystick would be much better to play could you add a joystick?

**Comment by qingjun1991** [comment]
> > 问题已经解决,请更新补丁
> 
> Good night, how are you? I hope so, I noticed a problem in dream factory, when installing the app on Android 11 it says that the app was made for an older version of Android and that it may not work correctly
> 
> A joystick would be much better to play could you add a joystick?

1、我做成了任意版本的安卓都能玩的，所以系统会提示这个消息，忽略它即可
2、为了能精确地控制人物，我做成了这样的；你也可以另外安装游戏键盘app，也可以外接游戏手柄

**Comment by Rafaelfilp** [comment]
> 



> > > 问题已经解决,请更新补丁
> > 
> > 
> > Good night, how are you? I hope so, I noticed a problem in dream factory, when installing the app on Android 11 it says that the app was made for an older version of Android and that it may not work correctly
> > A joystick would be much better to play could you add a joystick?
> 
> 1、我做成了任意版本的安卓都能玩的，所以系统会提示这个消息，忽略它即可 2、为了能精确地控制人物，我做成了这样的；你也可以另外安装游戏键盘app，也可以外接游戏手柄

thanks bro :) will there be more updates?

**Comment by qingjun1991** [comment]
> > 
> 
> > > > 问题已经解决,请更新补丁
> > > 
> > > 
> > > Good night, how are you? I hope so, I noticed a problem in dream factory, when installing the app on Android 11 it says that the app was made for an older version of Android and that it may not work correctly
> > > A joystick would be much better to play could you add a joystick?
> > 
> > 
> > 1、我做成了任意版本的安卓都能玩的，所以系统会提示这个消息，忽略它即可 2、为了能精确地控制人物，我做成了这样的；你也可以另外安装游戏键盘app，也可以外接游戏手柄
> 
> thanks bro :) will there be more updates?

现在只修复bug

**Comment by Rafaelfilp** [comment]
> > > 
> > 
> > 
> > > > > 问题已经解决,请更新补丁
> > > > 
> > > > 
> > > > Good night, how are you? I hope so, I noticed a problem in dream factory, when installing the app on Android 11 it says that the app was made for an older version of Android and that it may not work correctly
> > > > A joystick would be much better to play could you add a joystick?
> > > 
> > > 
> > > 1、我做成了任意版本的安卓都能玩的，所以系统会提示这个消息，忽略它即可 2、为了能精确地控制人物，我做成了这样的；你也可以另外安装游戏键盘app，也可以外接游戏手柄
> > 
> > 
> > thanks bro :) will there be more updates?
> 
> 现在只修复bug

Could you put inputbrige in dreamfectory and a virtual control for exagaer if you want, I'll send you a picture of it in gmail

**Comment by qingjun1991** [comment]
我网上查了一下，貌似你安装完InputBridge，就能用了，不需要我做什么吧？

另外你可以看看：
"!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.5、安卓版如何修改触屏按键.txt"
"!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.9、安卓版如何使用游戏键盘app.txt"
"!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.10、安卓版如何使用USB游戏手柄.txt"

**Comment by Rafaelfilp** [comment]
> 



> 我网上查了一下，貌似你安装完InputBridge，就能用了，不需要我做什么吧？
> 
> 另外你可以看看： "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.5、安卓版如何修改触屏按键.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.9、安卓版如何使用游戏键盘app.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.10、安卓版如何使用USB游戏手柄.txt"

Oh sorry I'll look later

**Comment by Rafaelfilp** [comment]
> 我网上查了一下，貌似你安装完InputBridge，就能用了，不需要我做什么吧？
> 
> 另外你可以看看： "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.5、安卓版如何修改触屏按键.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.9、安卓版如何使用游戏键盘app.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.10、安卓版如何使用USB游戏手柄.txt"

And that needs to execute a file and in dreamfactory it is not possible to execute

**Comment by qingjun1991** [comment]
> > 我网上查了一下，貌似你安装完InputBridge，就能用了，不需要我做什么吧？
> > 另外你可以看看： "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.5、安卓版如何修改触屏按键.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.9、安卓版如何使用游戏键盘app.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.10、安卓版如何使用USB游戏手柄.txt"
> 
> And that needs to execute a file and in dreamfactory it is not possible to execute

那就用不了InputBridge了，不过你可以看看那3个更简单的方案

**Comment by Rafaelfilp** [comment]
> > > 我网上查了一下，貌似你安装完InputBridge，就能用了，不需要我做什么吧？
> > > 另外你可以看看： "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.5、安卓版如何修改触屏按键.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.9、安卓版如何使用游戏键盘app.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.10、安卓版如何使用USB游戏手柄.txt"
> > 
> > 
> > And that needs to execute a file and in dreamfactory it is not possible to execute
> 
> 那就用不了InputBridge了，不过你可以看看那3个更简单的方案

Which?

**Comment by qingjun1991** [comment]
> > > > 我网上查了一下，貌似你安装完InputBridge，就能用了，不需要我做什么吧？
> > > > 另外你可以看看： "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.5、安卓版如何修改触屏按键.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.9、安卓版如何使用游戏键盘app.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.10、安卓版如何使用USB游戏手柄.txt"
> > > 
> > > 
> > > And that needs to execute a file and in dreamfactory it is not possible to execute
> > 
> > 
> > 那就用不了InputBridge了，不过你可以看看那3个更简单的方案
> 
> Which?

"!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.5、安卓版如何修改触屏按键.txt"
"!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.9、安卓版如何使用游戏键盘app.txt"
"!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.10、安卓版如何使用USB游戏手柄.txt"

**Comment by Rafaelfilp** [comment]
> 



> > > > > 我网上查了一下，貌似你安装完InputBridge，就能用了，不需要我做什么吧？
> > > > > 另外你可以看看： "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.5、安卓版如何修改触屏按键.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.9、安卓版如何使用游戏键盘app.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.10、安卓版如何使用USB游戏手柄.txt"
> > > > 
> > > > 
> > > > And that needs to execute a file and in dreamfactory it is not possible to execute
> > > 
> > > 
> > > 那就用不了InputBridge了，不过你可以看看那3个更简单的方案
> > 
> > 
> > Which?
> 
> "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.5、安卓版如何修改触屏按键.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.9、安卓版如何使用游戏键盘app.txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\1、说明、问题、教程\4.10、安卓版如何使用USB游戏手柄.txt"

Thanks

**Comment by NuggetFangus** [comment]
Will game dream factory someday have survival mode?

**Comment by qingjun1991** [comment]
> Will game dream factory someday have survival mode?
For flexibility, the game dream factory uses the method of adding code to implement different modes.

Examples:

Survival Mode:
"!GameDreamFactory\stages\bwBase1横版.def"

Survivors Mode:
"!GameDreamFactory\stages\bwBase1横版格斗幸存者.def"

LeadSoldier Mode:
"!GameDreamFactory\stages\bwBase1横版百兵.def"

Melee Mode:
"!GameDreamFactory\stages\bwBase1横版混战.def"

Tag Mode:
"!GameDreamFactory\stages\kfm宽屏Tag.def"

Turns Mode:
"!GameDreamFactory\stages\kfm宽屏车轮战.def"

**Comment by qingjun1991** [comment]
> Will game dream factory someday have survival mode?

For flexibility, the game dream factory uses the method of adding code to implement different modes.

Examples:

Survival Mode:
"!GameDreamFactory\stages\bwBase1横版.def"

Survivors Mode:
"!GameDreamFactory\stages\bwBase1横版格斗幸存者.def"

LeadSoldier Mode:
"!GameDreamFactory\stages\bwBase1横版百兵.def"

Melee Mode:
"!GameDreamFactory\stages\bwBase1横版混战.def"

Tag Mode:
"!GameDreamFactory\stages\kfm宽屏Tag.def"

Turns Mode:
"!GameDreamFactory\stages\kfm宽屏车轮战.def"

**Comment by NuggetFangus** [comment]
How can i make this code work in a personal mugen of mine?

**Comment by qingjun1991** [comment]
> How can i make this code work in a personal mugen of mine?

your mugen add "!GameDreamFactory\chars" and "!GameDreamFactory\stages", select.def add stages, then you can select stage to play

Detailed code tutorial:
"!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂"
